AddonBicepTemplate/AzureMonitorAlertsProfile.bicep

```diff
@@ -1,534 +0,0 @@
-param monitorWorkspaceName string
```

Comment:
this deleted file is currently referenced by https://aka.ms/azureprometheus-alerts-bicep in our docs....

Reply:
my bad, I deleted this by mistake. Adding back.